frontend/packages/pipelines-plugin/src/components/taskruns/TaskRunDetailsStatus.tsx

```diff
@@ -38,6 +39,10 @@ const TaskRunDetailsStatus = ({ taskRun }) => {
           </dd>
         </dl>
       )}
+      <WorkspaceResourceLinkList
+        workspaces={taskRun?.spec.workspaces}
```

Comment:
```suggestion
        workspaces={taskRun.spec?.workspaces}
```

Reply:
If the TaskRun is there, I'm pretty sure Spec _has_ to be there too... I don't think conditional checks are needed. You just have to support no workspaces (empty array or undefined) in the component.